src/chunkserver/datastore/file_pool.cpp

```diff
@@ -709,6 +876,12 @@ bool FilePool::ScanInternal() {
         }
     }
 
+    currentState_.chunkNum = tmpvec.size();
+    LOG(ERROR) << "chunkNum = " << currentState_.chunkNum;
+    currentState_.chunkNum += CountAllocatedNum(poolOpt_.copysetDir);
+    currentState_.chunkNum += CountAllocatedNum(poolOpt_.recycleDir);
+    LOG(ERROR) << "chunkNum = " << currentState_.chunkNum;
```

Comment:
remove these two `LOG(ERROR)`, or change to `LOG(INFO)`

Reply:
These are debug logs and I forgot to delete them.